extras/jsonrpc/daemon_types.go

```diff
@@ -321,6 +317,20 @@ type Claim struct {
 	ValidAtHeight           int              `json:"valid_at_height,omitempty"`
 }
 
+type Meta struct {
```

Comment:
Naming this something like `ClaimMeta` would be beneficial

Reply:
I would, but I think it more valuable to align with the response from the [SDK](https://lbry.tech/api/sdk#resolve) which uses `meta`.

Reply:
In JSON it's `meta` because it's already inside of the claim value, so it would be redundant to say `claim`. Since you're working with a flatter namespace you should definitely call it ClaimMeta here to not be ambiguous.

Reply:
@tiger5226 @sayplastic Also, just fyi, the metadata is sent from wallet server to the client via this protobuf, which is called `ClaimMeta`: https://github.com/lbryio/types/blob/master/v2/proto/result.proto#L22